ostd/src/arch/loongarch/mm/mod.rs

```diff
@@ -102,12 +103,17 @@ pub(crate) fn tlb_flush_all_including_global() {
     }
 }
 
+pub(crate) fn can_sync_dma() -> bool {
+    false
+}
+
 /// # Safety
 ///
-/// The caller must ensure that the virtual address range and DMA direction correspond correctly to
-/// a DMA region.
-pub(crate) unsafe fn sync_dma_range(_range: Range<Vaddr>, _direction: DmaDirection) {
-    unimplemented!("DMA synchronization is unimplemented in LoongArch64")
+/// The caller must ensure that the virtual address range and DMA direction
+/// correspond correctly to a DMA region.
+#[expect(clippy::extra_unused_type_parameters)]
+pub(crate) unsafe fn sync_dma_range<D: DmaDirection>(_range: Range<Vaddr>) {
+    todo!("Implement DMA synchronization for LoongArch64 architecture");
 }
```

Comment:
I think this might be a rebase error. In this case, I believe `unimplement!` is more appropriate than `todo!` Or is this intentional?